pytorch_pfn_extras/writing/_writer_base.py

```diff
@@ -89,6 +93,7 @@ def list(
             if path_or_prefix is None:
                 raise ValueError(
                     "'path_or_prefix' must not be None in recursive mode.")
+            path_or_prefix = self.get_actual_path(path_or_prefix)
```

Comment:
Need to normalize path even if the `recursive=False` case.

Reply:
fixed!